_posts/2017-10-31-linux-release-2017.10.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+style: style2
+title:  "Latest Debian / Mint / Ubuntu Release Now Available"
+description: We have changed the version numbers and future builds will have the format YYYY.MM
```

Comment:
The news post's content/description has nothing to do with the title.

Reply:
Thanks for spotting this - fixed!